scripts/tsconfig.json

```diff
@@ -4,5 +4,6 @@
     "module": "commonjs",
     "noImplicitAny": true,
     "strictNullChecks": true
-  }
+  },
+  "include": ["../src", "."]
```

Comment:
As an aside, it's curious that you had to add this.

This tsconfig.json is used whenever we run one of our scripts with e.g.:
```
ts-node-script -r tsconfig-paths/register scripts/vaccination_alerts/parse-vaccination-phases.ts
```

and it works in that context. 🤷

Reply:
yeah, that is weird.  I can't explain this.  though it stands to reason that a type-script script-running engine would compile typescript according to the tsconfig.json it finds.